pkg/juicefs/mount/builder/vci-serverless.go

```diff
@@ -64,13 +71,27 @@ func NewVCIBuilder(setting *config.JfsSetting, capacity int64, app corev1.Pod, p
 func (r *VCIBuilder) NewMountSidecar() *corev1.Pod {
 	pod := r.genCommonJuicePod(r.genNonPrivilegedContainer)
 	// overwrite annotation
-	pod.Annotations = map[string]string{
-		VCIPropagationConfig: VCIPropagationConfigValue,
-		VCIPropagation: fmt.Sprintf(`[{
-			"container": "jfs-mount",
-			"mountPath" : "%s"
-		}]`, r.jfsSetting.MountPath),
+	if pod.Annotations == nil {
+		pod.Annotations = make(map[string]string)
+	}
+	mountContainerName := r.genMountContainerName()
+	pod.Annotations[VCIPropagationConfig] = VCIPropagationConfigValue
+
+	propagations := make([]VCIPropagationStruct, 0)
+	propagations = append(propagations, VCIPropagationStruct{
+		Container: mountContainerName,
+		MountPath: r.jfsSetting.MountPath,
+	})
+	if v, ok := r.app.Annotations[VCIPropagation]; ok {
+		var vciPropagations []VCIPropagationStruct
+		if err := json.Unmarshal([]byte(v), &vciPropagations); err == nil {
+			propagations = append(propagations, vciPropagations...)
+		} else {
+			klog.Errorf("failed to unmarshal VCIPropagation annotation: %v", err)
```

Comment:
any change we print the original annotations string in this log?

```suggestion
			klog.Errorf("failed to unmarshal VCIPropagation annotation %s %v", v, err)
```

Reply:
emmm...I believe there is no need to print